forwards_paper.tex

```diff
@@ -335,16 +351,18 @@ \subsection*{Recording the pedigree in forwards time}
 For example, in Algorithm~\algref{W} we would generate mutations after
 we have recorded the edges joining the new node $n$ to its parents
 $P_a$ and $P_b$ in step \algref{W7}.
-For example, if we assume that infinite sites mutations
-occur at rate $\mu$ per generation then there are $\mbox{Poisson}\left((r - \ell)(\tau_p
+For example, if we assume that mutations occur according to the infinite sites model
+at rate $\mu$ per generation per unit length then there are 
+$\mbox{Poisson}\left(\mu (r - \ell)(\tau_p
 - \tau_c)\right)$ mutations on each edge $(\ell, r, p, c)$. Each of these mutations
-will occur on a distinct site $x$ drawn uniformly from $(\ell, r]$ and be over node $c$.
+will occur on a distinct site $x$ drawn uniformly from $(\ell, r]$ and be associated with node $c$.
 It is straightforward to record this information during the simulation, but it
 is significantly simpler and more efficient to generate these mutations
-after the simulation has completed. In the case of infinite sites mutations
-there is no difference between generating these mutations during the forwards
-in time simulation and after it has completed. We are using precisely the same
-information.
+after the simulation has completed. 
+% In the case of infinite sites mutations
```

Comment:
I don't think we need to make this point explicitly - mutations can be drawn exactly as before under any mutation model, but we don't want to get into this here, I think.